eng/pipelines/release-cli.yml

```diff
@@ -479,62 +497,68 @@ stages:
 
         strategy:
           matrix:
-            Windows:
-              Pool: azsdk-pool-mms-win-2019-general
-              OSVmImage: MMS2019
-              TestShell: cmd
-              TestCommand: >
-                cmd /c test-windows-install.cmd %BASEURL%
-            Linux:
+            LinuxSh:
```

Comment:
Is Sh the right name given you are testing the ps1 script here?

Reply:
Also of note: Investigate centos support matrix.